package com.sprint.mission.discodeit.exception.user;

import com.sprint.mission.discodeit.exception.ErrorCode;
import java.util.Map;
import java.util.Objects;
import java.util.UUID;

public class UserStatusAlreadyExistsException extends UserException {

  public UserStatusAlreadyExistsException(UUID userId) {
    super(ErrorCode.USER_STATUS_EXISTS, Map.of("userId", userId));
  }
}
